frame/proxy/src/lib.rs

```diff
@@ -250,11 +250,23 @@ pub mod pallet {
 			proxy_type: T::ProxyType,
 			delay: T::BlockNumber,
 		) -> DispatchResult {
-			let who = ensure_signed(origin)?;
-			let delegate = T::Lookup::lookup(delegate)?;
-			Self::remove_proxy_delegate(&who, delegate, proxy_type, delay)
+			
+			Self::remove_all_proxy_delegates(&who, delegate, proxy_type, delay)
+		}
+
+		
+		#[pallet::weight(T::WeightInfo::remove_all_proxy_delegates(T::MaxProxies::get()))]
+		pub fn pub remove_all_proxy_delegates(origin: OriginFor<T>,
+			delegate: AccountIdLookupOf<T>,
+			proxy_type: T::ProxyType,
+			delay: T::BlockNumber,
+		)-> DispatchResult{
+				let who = ensure_signed(origin)?;
```

Comment:
You just moved the logic to a new call?

Reply:
Yes the original instructions given me to move the logic to new method and call from remove_proxy.
[https://github.com/paritytech/substrate/issues/7557](Please read the instructions for 7557)

Please let me know what changes are required for this issue.

Reply:
@vjgaur Sorry but I think you misunderstood. The intention is to move to a new _method_, but what you did here is move the function body to a new _extrinsic_. Please familiarize yourself with the difference between the two.

Reply:
so as I understand I need to move the code of remove_proxy extrinsic to a new method remove_all_proxy_delegates which will be outside the #[pallet::call] since we need a method not extrinsic function and later call this new method from remove_proxy extrinsic. Am I correct ? 
Pardon me If I ask any dumb question I am just getting started

Reply:
Yes, that sounds correct.

Reply:
@KiChjang I pushed the changes with all test passed. Please review and let me know if any change required

Reply:
@ggwpez Can you please review the PR and share me your feedback ?